packages/data-transport-layer/src/services/run.ts

```diff
@@ -48,8 +47,8 @@ interface Bcfg {
         'legacySequencerCompatibility',
         false
       ),
-      stopL2SyncAtBlock: config.uint('stopL2SyncAtBlock'),
       sentryDsn: config.str('sentryDsn'),
+      defaultSource: config.str('defaultSource', 'l1'),
```

Comment:
do you mean to make the default `'batched'` here instead of `'l1'`?

Reply:
Good catch, this is the bug that is causing CI to fail